includes/payment-methods/class-wc-stripe-upe-payment-method.php

```diff
@@ -59,10 +52,8 @@ abstract class WC_Stripe_UPE_Payment_Method {
 
 	/**
 	 * Create instance of payment method
-	 *
-	 * @param WC_Payments_Token_Service $token_service Instance of WC_Payments_Token_Service.
 	 */
-	public function __construct( $token_service ) {
+	public function __construct() {
 		$main_settings       = get_option( 'woocommerce_stripe_settings' );
```

Comment:
```suggestion
		$main_settings = get_option( 'woocommerce_stripe_settings' );
```

Reply:
Whoops, this was ugly. Applied your suggestion in 181780cfbe22e7ab11cb9515ca4f857196e8c951.